kitchen.validate.yml

```diff
@@ -40,7 +40,6 @@ suites:
         - write_common_udev_configuration_files
         - ec2blkdev_service_installation
         - debian_udevd_reload_configuration
-        - package_repos
```

Comment:
Why did you remove it? 

Is it needed only for intel_mpi? If this is not the case I prefer having it explicitly called.

Reply:
No, I have removed it because that InSpec test has been remove in previous PR